docs/packages-license.md

```diff
@@ -1908,6 +1908,17 @@ LGPL-3.0-or-later permitted
 
 
 
+<a name="@radix-ui/react-radio-group"></a>
+### @radix-ui/react-radio-group v1.2.2
+#### 
+
+##### Paths
+* /home/runner/work/liam/liam
+
+<a href="http://opensource.org/licenses/mit-license">MIT</a> permitted
+
+
+
```

Comment:
Although this diff is already in the main branch, merging this PR shouldn’t add any duplicate entries.